ci_framework/roles/os_must_gather/tasks/main.yml

```diff
@@ -61,7 +61,9 @@
           oc adm must-gather --image {{ cifmw_os_must_gather_image }}
           --timeout {{ cifmw_os_must_gather_timeout }}
           --host-network={{ cifmw_os_must_gather_host_network }}
-          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs > {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
+          --dest-dir {{ cifmw_os_must_gather_output_dir }}/logs
+          -- ADDITIONAL_NAMESPACES={{ cifmw_os_must_gather_additional_namespaces }} gather &> {{ cifmw_os_must_gather_output_dir }}/logs/os_must_gather.log
+
   rescue:
     - name: Inspect the cluster after must-gather failure
```

Comment:
@fmount I know this was already kind of meh and this is not you fault, but can you please make this task ansible friendly? I mean, split it two:

1.  One task that creates the `{{ cifmw_os_must_gather_output_dir }}/logs/oc_inspect` directory
2. Another one, that using a loop over `cifmw_os_must_gather_additional_namespaces | split(',')` runs `oc adm inspect namespace/"${ns}" --dest-dir={{ cifmw_os_must_gather_output_dir }}/logs/oc_inspect` You can set the ignore errors to true, that way if it fails, we can get the entire output of the failure, instead of relying on a dumb message like `echo "Error getting logs from ${ns}"`.

Reply:
Doing that and refining in a couple of iterations